staging/kubefed/Chart.yaml

```diff
@@ -1,9 +1,9 @@
-apiVersion: v1
+apiVersion: v2
 description: KubeFed helm chart
 name: kubefed
-version: 0.0.4
-appVersion: v0.1.0-rc6
-home: https://github.com/kubernetes-sigs/kubefed
-maintainers:
-  - name: s12chung
-  - name: alejandroEsc
+version: 0.0.5
```

Comment:
Think you need to package and add to `gh-pages`. 0.0.5 is missing https://github.com/mesosphere/charts/tree/gh-pages/staging

which is probably why tests are failing.

Reply:
naw, its a requirements field issue, fixing now. Also helm2 will fail on purpose.